packages/autocomplete-js/src/__tests__/autocomplete.test.ts

```diff
@@ -1,6 +1,5 @@
-import { fireEvent } from '@testing-library/dom';
-
-import { autocomplete } from '../autocomplete';
+import { autocomplete } from '@algolia/autocomplete-js';
```

Comment:
Why did you need to change this import?

It'll now require to build the project to run these tests.

Reply:
That's a mistake while I was struggling with dom testing!